plugin-packages/multimedia/README.md

```
@@ -0,0 +1,25 @@
+```
+
+## Development
+
+### Getting Started
+
+``` bash
+# demo
+pnpm --filter @galacean/effects-plugin-multimedia dev
+```
+
+> Open in browser: http://localhost:8081/demo/
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)